### PR TITLE
Add election command

### PR DESCRIPTION
**Pull request by windowsboy111**
We need #1 to be merged first to apply the election command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # 0w0nator
 The discord bot for Siri's World
+
+**Join Siri's World: [https://bit.ly/siriw](https://discord.gg/8SUAKBkpDR)**
```
Comment (TNTprizz) [inline README.md:4]: hummm
I think no one will copy the bitly link......
so just use `[here](https://discord.gg/8SUAKBkpDR)`

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+pyTableMaker
```
Comment (TNTprizz) [inline requirements.txt:1]: you know what? I just applied the requirements.txt into the config.yml
so ofc it is your rights that you use requirements.txt but I think yml better

File: src/cogs/election.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            result = line + "\n"
+            continue
+        result += line + "\n"
+    if result != "":
+        results.append(result)
+    return results
+
+
+
+def setup(bot):
```
Comment (TNTprizz) [inline src/cogs/election.py:21]: you know what is a class?
you can simply import a cog using a command class like mine
so that import cogs become easier

**Review by TNTprizz — COMMENTED** [review]
you may use config.yml instead of requirements.txt cuz the folder will become complex which is not good for siri I think

**Comment by TNTprizz** [comment]
> 
> 
> We need #1 to be merged first to apply the election command.

erm no need #1 first but #3 first?

File: README.md (hunk 1 above)
Comment (windowsboy111) [reply README.md:4]: I thought people could like memorise the link easily so I put it there.

File: src/cogs/election.py (hunk 3 above)
Comment (windowsboy111) [reply src/cogs/election.py:21]: The thing is I don't want a Cog on this command because I am not sure what name I should give to the cog.

Comment (TNTprizz) [reply src/cogs/election.py:21]: Same as file name is ok becuz it is just a name

File: README.md [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+======
 - The discord bot for Siri's World
 - Developer team is named as Lazy Programming Developer Factory(LPDF)
 
 - Important:
-> Configure all your stuffs in file "config.yml" and add indexes if you want
\ No newline at end of file
+> Configure all your stuffs in file "config.yml" and add indexes if you want
+
+- `wb/electionCommand` & `master`
```
Comment (TNTprizz) [inline README.md:18]: conflict resolved

**Review by TNTprizz — APPROVED** [review]
resolved conflict about README.md

**Pull request merged**